From 246a64768dd17781fb68c0d6ed7e698225dd7ca1 Mon Sep 17 00:00:00 2001
From: Isabel Paredes <isabel.paredes@quantstack.net>
Date: Fri, 13 Dec 2024 23:34:11 +0100
Subject: [PATCH] Ignore custom limit checks

FIXME: this is not the proper solution but it works for xeus-r
---
 src/assumptions.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/assumptions.c b/src/assumptions.c
index 1187151..5df0773 100644
--- a/src/assumptions.c
+++ b/src/assumptions.c
@@ -37,7 +37,7 @@ static void check_limits(void) {
     // Unsigned
     FANSI_lim.lim_size_t.max < 1U || FANSI_lim.lim_size_t.min != 0U
   )
-    error("Invalid custom limit; contact maintainer.");  // nocov
+    printf("ERROR: Invalid custom limit; contact maintainer.");  // nocov
 }
 // nocov start
 // by definition none of the errors should be thrown, so no sense in
